recipes/alphashape/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set name = "alphashape" %}
+{% set version = "1.0.1" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 4f83279dfccc71c17908161ddfa64f3c56727c10022179f7edec70857c636088
+
+build:
+  number: 0
+  noarch: python
+  entry_points:
+    - alphashape=alphashape.cli:main
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - click >=6.0
+    - click_log >=0.3.2
+    - pip
+    - python
+    - scipy >=1.0.0
+    - shapely >=1.4.0
+  run:
+    - click >=6.0
+    - click_log >=0.3.2
```

Comment:
The name of the conda package is `click-log`, and not `click_log`, so please fix the naming accordingly.

Reply:
fixed.